newscoop.conf

```diff
@@ -0,0 +1,13 @@
+<VirtualHost *:80>
+      DocumentRoot /var/www/newscoop
+      ServerName docker.newscoop
```

Comment:
well, problem is that on mac os docker does not run on localhost, but rather in a virtual machine with its own IP.  I've tested build with ServerName localhost, but it still requires an /etc/hosts update on mac os for the boot2docker ip, because localhost is the mac os machine, not the docker client.

Not sure the best way to solve this problem.